egs/chime5/s5/local/run_wpe.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# Copyright 2018 Johns Hopkins University (Author: Aswin Shanmugam Subramanian)
```

Comment:
What python was this tested with?  You could consider invoking as python2 or python3 (python3 better, but only if you can confirm it works).